embrace_mobile/assets/dashboards/embrace_mobile_overview.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
If this is using our "Screenboard," we recommend upgrading this to use our "Dashboard" instead. Thanks!